spikeinterface/extractors/neoextractors/neobaseextractor.py

```diff
@@ -203,13 +203,13 @@ def _auto_guess_sampling_frequency(self):
 
 
 class NeoSortingSegment(BaseSortingSegment):
-    def __init__(self, neo_reader, segment_index, use_natural_unit_ids, t_start, sampling_freq):
+    def __init__(self, neo_reader, segment_index, use_natural_unit_ids, t_start, _sampling_frequency):
```

Comment:
why the `_`? It's an init argument, I'd make it just `sampling_frequency`

Reply:
Good catch, it was a mistake.